# <mayhem_program::state::GlobalParams as anchor_lang::AccountDeserialize>::try_deserialize  (yours 1000 B at .text+0x23dc0; original 1000 B at .text+0x675d8)
# distance 1 instructions; evidence: called-by anchor_lang::accounts::account::Account<T>::try_from#27dc; same-callees 7/7
--- original
+++ yours
@@ -48,7 +48,7 @@
 stxdw [r10-224], r1
 stw [r10-80], 3002
 stb [r10-152], 2
-stw [r10-208], 12
+stw [r10-208], 27
 stdw [r10-216], 36
 stdw [r10-232], 0
 mov64 r1, r10
